awswrangler/s3/_read_parquet.py

```diff
@@ -726,74 +518,65 @@ def read_parquet(
     >>> df = wr.s3.read_parquet(path, dataset=True, partition_filter=my_filter)
 
     """
-    session: boto3.Session = _utils.ensure_session(session=boto3_session)
     paths: List[str] = _path2list(
```

Comment:
This is out of the scope of this PR, but is there any value to adding distributed support to `list_objects` which supports `_path2list()` under the hood? I noticed it wasn't on the roadmap.

Reply:
Yes, I think if there is a way to distribute it, we definitely should. Feel free to create an issue and take it :)